File: MtgSearch/MtgSearch.Server/Models/Api/LanguageSpec.cs
namespace MtgSearch.Server.Models.Api
{
    public class LanguageSpec
    {
        public List<FunctionDefinition> FunctionDefinitions { get; set; }
        public ExpressionSpec Expressions { get; set; }
        public ExpressionSpec IsExpressions { get; set; }
    }
}

File: MtgSearch/mtgsearch.client/src/app/models/LanguageSpec.ts
import { ExpressionSpec } from "./ExpressionSpec";
import { FunctionDefinition } from "./FunctionDefinition";

export class LanguageSpec {
  public functionDefinitions: FunctionDefinition[] = [];
  public expressions: ExpressionSpec | undefined;
  public isExpressions: ExpressionSpec | undefined;
}
